### DISCUSSION TITLE
Pagina de la pastelería

### DISCUSSION THREAD

**Discussion by Josue-HernandezL** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#154] [Q&A]
**### Mi duda o pregunta es, si donde dice, ver los sabores y/o precios, se refiere a que pongamos imagenes o solo a los textos?**

> - El cliente de la pastelería necesita ver los diferentes sabores de pasteles, así como los precios de cada uno.
> - El cliente de la pastelería necesita ver los diferentes adornos con los que se puede decorar el pastel y los precios de cada uno.

**Comment by GloriaCoralCerecedo** [comment]
Podría ser solo texto, igual imágenes seria genial, recuerda que no hay limite, puedes hacer lo que gustes.

**Reply by ahranza** [reply]
Hola! Disculpa, ¿Cómo es para poder poder seleccionar varias opciones al mismo tiempo? Usé input radio pero sólo me deja elegir una.

**Reply by JLuisPrz** [reply]
Hola, es con input type = "checkbox"

**Reply by ahranza** [reply]
muchas gracias!

**Comment by JuanSalvador73** [comment]
¿Donde se menciona eso de la página de los pasteles, que no era una de tacos según la sesión del jueves? Respecto a tu pregunta, lo ideal es incluir fotos de los pasteles y precios, puede ser en una tabla o con uso de divs - eso se verá mas adelante - Puedes checar el sitio de la pastelería "El Globo" como referencia o el de "La Esperanza". Saludos.

**Reply by JuanSalvador73** [reply]
Ya ví la práctica de la pastelería. :D

**Comment by DanyVeneno** [comment]
Hola Josue imaginate como te gustaria a ti visitar un sitio en donde vas a comprar un pastel entre más datos me des para que yo te compre seria un parametro o como dicen de la vista nace el amor. Saludos Buena tarde
